src/AppState.ts

```diff
@@ -101,9 +107,11 @@ function cleanCollectable(input: { [key: string]: any }): ICollectable {
   const key = name
     .toLowerCase()
     .replace(/ /g, "_")
-    .replace("'", "")
-    .replace("-", "_")
-    .replace(".", "");
+    .replaceAll("'", "")
```

Comment:
We can't use `replaceAll`, this is what is crashing the site. You either need to add a polyfill or use replace with regex and a global flag (see above). We have enough cases we should probably just merge them into a regex

Reply:
Ah, good catch.  I must've just gotten "lucky" that I was on Firefox 😂  I'll just swap this out with a regex for global replace in a cross-browser compliant way, my bad.

Reply:
Should be fixed now (using global regex replace)